server/src/models/backenderror.ts

```diff
@@ -0,0 +1,9 @@
+export class BackendError {
```

Comment:
Nice idea! But I would not propagate the whole stacktrace to the client. Maybe we should log the errors to a errorlog and only provide a clear message to the client. Additionally we should collect the messages in a central location (object, file etc.)

I would open a separate issue for that